Source/makefile

```diff
@@ -30,23 +30,29 @@ TYPE= SHARED
 #	if two source files with the same name (source.c or source.cpp)
 #	are included from different directories.  Also note that spaces
 #	in folder names do not work well with this makefile.
-SRCS= BeWake.cpp	\
-	ConfigView.cpp	\
-	HueSlider.cpp	\
-	Utils.cpp
+SRCS =	\
```

Comment:
Please use a current makefile_engine makefile (In Tracker > New > Makefile).
In front of every "\" just one space.

Reply:
This is still an old makefile.

Reply:
@humdingerb Where is that makefile?

Reply:
@humdingerb But I didn't find that file.

Reply:
I mean that I didn't find the option of "new"

Reply:
Right-click the Desktop. It's the second item in the context menu.

Reply:
@humdingerb I am getting this problem when switched to new makefile - 
```mkdir -p objects.x86-cc2-release; \
mkdepend  -I./  -Iobjects.x86-cc2-release/ -p .cpp:objects.x86-cc2-release/%n.o -m -f "objects.x86-cc2-release/Utils.d" Utils.cpp
mkdir -p objects.x86-cc2-release; \
mkdepend  -I./  -Iobjects.x86-cc2-release/ -p .cpp:objects.x86-cc2-release/%n.o -m -f "objects.x86-cc2-release/HueSlider.d" HueSlider.cpp
mkdir -p objects.x86-cc2-release; \
mkdepend  -I./  -Iobjects.x86-cc2-release/ -p .cpp:objects.x86-cc2-release/%n.o -m -f "objects.x86-cc2-release/ConfigView.d" ConfigView.cpp
mkdir -p objects.x86-cc2-release; \
mkdepend  -I./  -Iobjects.x86-cc2-release/ -p .cpp:objects.x86-cc2-release/%n.o -m -f "objects.x86-cc2-release/BeWake.d" BeWake.cpp
g++ -c BeWake.cpp  -I./  -Iobjects.x86-cc2-release/ -I-   -O3    -o "objects.x86-cc2-release/BeWake.o"
g++ -c ConfigView.cpp  -I./  -Iobjects.x86-cc2-release/ -I-   -O3    -o "objects.x86-cc2-release/ConfigView.o"
g++ -c HueSlider.cpp  -I./  -Iobjects.x86-cc2-release/ -I-   -O3    -o "objects.x86-cc2-release/HueSlider.o"
g++ -c Utils.cpp  -I./  -Iobjects.x86-cc2-release/ -I-   -O3    -o "objects.x86-cc2-release/Utils.o"
cat BeWake.rdef | cc -E  -I./  -Iobjects.x86-cc2-release/ -I-   -O3    - | grep -av '^#' | rc -I ./ -o "objects.x86-cc2-release/BeWake.rsrc" -
cc -o "objects.x86-cc2-release/BeWake"  objects.x86-cc2-release/BeWake.o objects.x86-cc2-release/ConfigView.o objects.x86-cc2-release/HueSlider.o objects.x86-cc2-release/Utils.o   -L./  -Lobjects.x86-cc2-release/    -lbe  -lscreensaver 
/boot/system/develop/lib/start_dyn.o: In function `_start':
start_dyn.c:(.text+0x57): undefined reference to `main'
collect2: ld returned 1 exit status
/boot/system/develop/etc/makefile-engine:246: recipe for target 'objects.x86-cc2-release/BeWake' failed
make: *** [objects.x86-cc2-release/BeWake] Error 1```

Reply:
Wait, it is fixed.